src/uct/sm/mm/posix/mm_posix.c

```diff
@@ -703,11 +703,11 @@ UCT_MM_TL_DEFINE(posix, &uct_posix_md_ops, uct_posix_rkey_unpack,
                  uct_posix_rkey_release, "POSIX_",
                  uct_posix_iface_config_table);
 
-UCT_TL_INIT(posix)
+UCS_F_CTOR UCT_TL_INIT(posix)
```

Comment:
why not call it from uct_init()? posix , sysv , cma are built-in transports

Reply:
> why not call it from uct_init()? posix , sysv , cma are built-in transports

partially correct: posix and sysv are built-int, cma, knem and xpmem are external modules:
```
> ls ucx/lib/ucx/
libucs_fuse.a   libucs_fuse.so.0      libuct_cma.la    libuct_cma.so.0.0.0  libuct_ib.so        libuct_knem.a   libuct_knem.so.0      libuct_rdmacm.la    libuct_rdmacm.so.0.0.0
libucs_fuse.la  libucs_fuse.so.0.0.0  libuct_cma.so    libuct_ib.a          libuct_ib.so.0      libuct_knem.la  libuct_knem.so.0.0.0  libuct_rdmacm.so
libucs_fuse.so  libuct_cma.a          libuct_cma.so.0  libuct_ib.la         libuct_ib.so.0.0.0  libuct_knem.so  libuct_rdmacm.a       libuct_rdmacm.so.0
```

fixed